### ISSUE TITLE
get_saq_sites curl error on Linux

### ISSUE THREAD

**Issue by BlaiseKelly** [skgrange/saqgetr#7]
Calling get_saq_sites on a Linux machine returns the following error:

Error in curl::new_handle() : An unknown option was passed in to libcurl

Replacing with a direct call of the table using the data.table function fread gets round it:
fread('http://aq-data.ricardo-aea.com/R_data/saqgetr/helper_tables/sites_table.csv.gz')

But any ideas what is causing the initial error?

**Comment by skgrange** [comment]
My immediate thought would be that your libcurl version is old, and some R functions, probably in the httr package, are passing extra arguments/options which are not being recognised. What does `curl::curl_version()$version` return and how old is your Linux distribution?

**Comment by BlaiseKelly** [comment]
Version 7.61.0

Anyway not a huge issue with work around, but just thought would flag in case others get the same problem.

Thanks for a superb package.

**Comment by skgrange** [comment]
Version 7.61.0 of libcurl was released in May, 2018 which is probably where to look if the issue needs to be resolved. I have checked the servers I have access to, and my own Ubuntu-based workstation, and these systems are running Version 7.68.0 which was released in January, 2020. 

I have not encountered this issue before, so I will leave this issue open for a month or so and see if it comes up again. If so, I will have a good look on what the best action is to address the issue. I hope the package still continues to help you out!
